book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] You did a great job setting up the classes and the associations between them. :+1: I have not seen anywhere in the code where the `Rental` class is getting instantiated. One place to do so is inside the `add_rental` method. You can make this method accept `date` and `person` parameters and create the `rental`. The following can be a possible solution. 😎️

**Example**
```
def add_rental(date, person)
  Rental.new(date, self, person)
end
```
   We are adding the rental to the book rentals and person rentals when it is initialized, so there is no need add it again inside this method. 🤝️